seerep-srv/seerep-core-fb/src/core-fb-conversion.cpp

```diff
@@ -102,6 +102,37 @@ seerep_core_msgs::DatasetIndexable CoreFbConversion::fromFb(const seerep::fb::Po
   return dataForIndices;
 }
 
+seerep_core_msgs::DatasetIndexable CoreFbConversion::fromFb(const seerep::fb::PointCloud2& cloud)
+{
+  seerep_core_msgs::DatasetIndexable dataForIndices;
+  fromFbDataHeader(cloud.header(), dataForIndices.header, seerep_core_msgs::Datatype::PointCloud);
+
+  // set bounding box for images to 0. assume no spatial extent
+  dataForIndices.boundingbox.min_corner().set<0>(0);
+  dataForIndices.boundingbox.min_corner().set<1>(0);
+  dataForIndices.boundingbox.min_corner().set<2>(0);
+  dataForIndices.boundingbox.max_corner().set<0>(0);
+  dataForIndices.boundingbox.max_corner().set<1>(0);
+  dataForIndices.boundingbox.max_corner().set<2>(0);
```

Comment:
It's possible but pretty ugly since we don't wont to iterate over the x,y,z twice. See https://github.com/agri-gaia/seerep/pull/193/commits/3187fcd24bf368e6dbddc64b0165c6e0b8881f05

Reply:
Issue for refactor in https://github.com/agri-gaia/seerep/issues/195